### ISSUE TITLE
Labels of the DiffraNet dataset

### ISSUE THREAD

**Issue by aandvalenzuela** [arturluis/diffranet#6]
Hi!

I was wondering were do you store the labels of the data. I have seen they can be accessed by using the torch.utils.data.DataLoader class, but I would like to implement my own method to load data and labels. I have also seen that the data under the /synthetic folder correspond only to three classes, why not the 5 stated in the article?

Many thanks in advance,
Andrea.

**Comment by arturluis** [comment]
Hello Andrea,

PyTorch's DataLoader infers the class based on the directory the data is in. I.e. the data is separated into 5 directories, one for each class, as you can see here:
,
![image](https://user-images.githubusercontent.com/5957286/156770860-40d11076-abe0-4b5d-ad8d-ee8e85245fe1.png)

The classes correspond to:
1 - blank
2 - no-crystal
3 - weak
4 - good
5 - strong

Note that the synthetic data is also separated into training, validation, and test sets, that's why there are three folders in `diffranet/data/synthetic/`. If you look into these 3 folders, you'll see that the data in there is split in the five classes mentioned above.

![image](https://user-images.githubusercontent.com/5957286/156770603-d02491d3-3c36-4b33-945f-5f1ef1c672cb.png)
